library/catalog/forms.py

```diff
@@ -49,8 +49,8 @@ def clean(self):
 
 
 class EditProfileForm(Form):
-    #user = MongoUser.objects(id='606ecd74e5fd490b3c6d0657').first()  # for autofill data session user
-#    user_id = CharField(required=False, widget=TextInput(attrs={'value': user.id}))
+    # user = MongoUser.objects(id='606ecd74e5fd490b3c6d0657').first()  # for autofill data session user
+    # user_id = CharField(required=False, widget=TextInput(attrs={'value': user.id}))
```

Comment:
it's part of the editing function
I took this from the developer
I return this to the developer